obstacle_stop_planner_refine/config/vehicle_info.param.yaml

```diff
@@ -0,0 +1,11 @@
+/**:
+  ros__parameters:
+    wheel_radius: 0.39
+    wheel_width: 0.42
+    wheel_base: 2.74 # between front wheel center and rear wheel center
+    wheel_tread: 1.63 # between left wheel center and right wheel center
+    front_overhang: 1.0 # between front wheel center and vehicle front
+    rear_overhang: 1.03 # between rear wheel center and vehicle rear
+    left_overhang: 0.1 # between left wheel center and vehicle left
+    right_overhang: 0.1 # between right wheel center and vehicle right
+    vehicle_height: 2.5
```

Comment:
Why did you add this? I think this should be removed.

Reply:
@kenji-miyake This file is used in launch_test.  https://github.com/tier4/obstacle_stop_planner_refine/pull/3/files/490178f7121c7286a9b5c78b2e0972b1c5331c8e#diff-231da54f78ec20658625ba7e58ff809ce59afdb8ec33db37b88789b53f901567R44

Note: I intend to use this file only for testing.

Reply:
Then I think you should name it as `test_vehicle_info.param.yaml` or move under `test/`.